packages/styles/scss/themes/expressive/components/_tag.scss

```diff
@@ -16,6 +16,74 @@
 /// @group tag-expressive
 @mixin tag-expressive {
   .#{$prefix}--tag {
+    @include carbon--type-style('body-long-01');
```

Comment:
I don't think we want to change the type token to `body-long-01` here because it's increasing the text size to 16px?

I thought the expressive theme token updates should have updated the original token `label-01` to 14px per the design specs, unless I am mistaken.